README.md

```diff
@@ -1 +1,5 @@
-# project-tabular-data-version
\ No newline at end of file
+# Project-tabular-data-version
+
+Experiment tracking and dataset versioning using Neptune.
```

Comment:
`Example of using XGBoost with Neptune for tracking meta-data and dataset versioning`?

I think it would be good to mention that this repo showcases XGBoost integration. Wdyt?